README.md

```diff
@@ -1,3 +1,40 @@
 # Karnau
 A puzzle game for iOS based on Karnaugh's manual method of simplifying boolean expressions.
 An implementation of the Quine-McCluskey simplification algorithm is provided.
+
+## Tutorial
+- Install and run on macOS with XCode.app
+- Press 'Start'
+- Select number of variables, either AB, ABC, ABCD, and a difficulty.
+- Tap and drag the mouse to select squares in multiples of two. 
+- To wrap around to the other table, tab and drag above or below the square and drag the mouse over the desired sister squares.
+- Press 'Check' to validate the solution.
+
+<img src="./img/img.6.25.11.png" width="200" alt="1"><img src="./img/img.6.25.13.png" width="200" alt="2"><img src="./img/img.6.25.21.png" width="200" alt="3"><img src="./img/img.6.25.24.png" width="200" alt="4"><img src="./img/img.6.25.46.png" width="200" alt="5">
+
+
+## Install and run on macOS
+- Install XCode.app from the App Store
+- Open Terminal.app in Applications > Utilities > Terminal.app
+- Install Developer Tools by typing gcc and accepting the installation prompts. ```gcc```
+- Clone this repository
+    ```
+    mkdir respositories
+    cd repositories
+    git clone https://github.com/lexlevi17/karnau
```

Comment:
They might not have git installed on their machine (or, gasp, not know what git is). Maybe tell them here that they should install OS X Command Line tools?